locksmith/src/middlewares/signatureValidationMiddleware.ts

```diff
@@ -1,10 +1,10 @@
 import sigUtil from 'eth-sig-util'
 import ethJsUtil from 'ethereumjs-util'
+import { Request, Response } from 'express-serve-static-core' // eslint-disable-line no-unused-vars, import/no-unresolved
```

Comment:
Is this around the use of Response?

Reply:
Eslint thinks that when you import types, they always go unused. So we have to disable that rule whenever we import types.